src/otserv.cpp

```diff
@@ -63,6 +64,19 @@ std::mutex g_loaderLock;
 std::condition_variable g_loaderSignal;
 std::unique_lock<std::mutex> g_loaderUniqueLock(g_loaderLock);
 
+/**
+ * It is preferable to keep the close button off as it closes the server without saving  (within five seconds)
+ * Choose to use "CTROL + C" or "CTROL + BREAK" for security close
+ * To activate/desactivate window
+ * \param MF_GRAYED Disable the "x" (force close) button
+ * \param MF_ENABLED Enable the "x" (force close) button
+*/
+void disableForceCloseButton() {
```

Comment:
would be nice if you move this function to the end of the file, since it's only a auxiliar function of the main flow

Reply:
The problem is that the compiler doesn't find the function if it sends it down.